sdk/docs/iot/README.md

```diff
@@ -36,7 +36,6 @@ The library targets made available via CMake are the following:
 
 - `az::iot::hub` - For Azure IoT Hub features ([API documentation here](https://azuresdkdocs.blob.core.windows.net/$web/c/az_iot/1.0.0/az__iot__hub__client_8h.html)).
 - `az::iot::provisioning` - For Azure IoT Provisioning features ([API documentation here](https://azuresdkdocs.blob.core.windows.net/$web/c/az_iot/1.0.0/az__iot__provisioning__client_8h.html)).
-- `az::iot::pnp` - **[BETA]** For Azure IoT Plug and Play features ([feature branch here](https://github.com/Azure/azure-sdk-for-c/tree/feature/iot_pnp)).
```

Comment:
Does this close the https://github.com/Azure/azure-sdk-for-c/issues/1825 or are there more things left to clean up there?

Reply:
If there's anything else from #1825, I couldn't find it.  I sent a fair amount of time yesterday trying to find broken stuff like this.